File: bovedaelectronicarecepcion/BERecepcion.Core/OrdenSurtimiento/Dto/SupplyOrder.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BERecepcion.Core.OrdenSurtimiento.Dto
{
    public class SupplyOrder
    {
        public Guid SupplyOrderId { get; set; }
        public string OrganismId { get; set; }
        public string Contract { get; set; }
        public string DocumentType { get; set; }
        public string SapOrder { get; set; }
        public string SiafOrder { get; set; }
        public string Type { get; set; }
        public string Creditor { get; set; }
        public string CreditorNumber { get; set; }
        public string CreditorRfc { get; set; }
        public string Total { get; set; }
        public string Currency { get; set; }
        public string MadeBy { get; set; }
        public string Representative { get; set; }
        public string Signer { get; set; }
        public string EsTRI { get; set; }
        public string Historical { get; set; }
        public DateTime ReceptionDate { get; set; }
    }
}

File: BER.Front/features/orden-surtimiento/model/supply-order.ts
import type { DateHelperModel } from './common'

export interface SupplyOrderDto {
  supplyOrderID: string
  organismID: string
  contract: string
  documentType: string
  sapOrder: string
  siafOrder: string
  type: string
  creditor: string
  creditorNumber: string
  creditorRFC: string
  total: string
  currency: string
  madeBy: string
  representative: string
  signer: string
  esTRI: boolean | null
  historical: boolean | null
  receptionDate: string | null

  administratorEmail: string
  administratorEmailSendDate: string | null
  administratorEmailReason: string
  administratorSignDate: string | null

  functionaryEmail: string
  functionaryEmailSendDate: string | null
  functionaryEmailReason: string
  functionarySignDate: string | null
  functionaryNotifyPemexDate: string | null
  functionaryFicha: string

  providerEmail: string
  providerEmailSendDate: string | null
  providerEmailReason: string
  providerSignDate: string | null
  providerNotifyPemexDate: string | null

  isFullSigned: boolean | null
  isCancel: boolean | null
  cancelDate: string | null
  cancelBy: string
  liberacionVPDate: string | null

  organismName: string
  organismClave: string
  functionarySignerName: string
  providerSignerName: string
  functionaryCancelName: string
  seguimiento: DateHelperModel[]
}
